### ISSUE TITLE
warning: localhost: not a Raspberry Pi

### ISSUE THREAD

**Issue by bgphoenix** [ebaauw/homebridge-rpi#99]
Device: Raspberry Pi 3
Homebridge 1.3.4
Node JS: v14.17.0

As per the steps in the installation readme, I am able to get get the pigs services running and I'm able to check the device hwver, but I keep getting the same error 'warning: localhost: not a Raspberry Pi'. 

I ran `sudo pigpiod -n localhost` to allow localhost connections

using the CLI rpi also I am unable to connect to localhost. I also tried 127.0.0.1 with no luck as mentioned in other issue threads. What am I missing ?

**Comment by ebaauw** [comment]
What's the output of `rpi -DD info`?

**Comment by fghekman** [comment]
I'm getting the same error 'Warning: localhost: not a Raspberry Pi'. After updating my existing setup from v1.3.2 to v1.3.3 of homebridge-rpi. After this message the plug-in stops working and my devices (created within homebridge-rpi) are deleted in the homekit setup.

I'm running it on a Raspberry Pi 3
Homebridge 1.3.4
Node JS: v14.17.0

**Comment by fghekman** [comment]
This is the output I get when running the command. I did revert back to version 1.3.2 at the moment.
```
~ $ rpi -DD info
rpi info: read file /proc/cpuinfo
rpi info: read file /proc/1/sched
rpi info: read file /sys/class/leds/led1/brightness
rpi info: exec vcgencmd measure_temp
rpi info: exec vcgencmd measure_clock arm
rpi info: exec vcgencmd measure_volts
rpi info: exec vcgencmd get_throttled
{
  "id": "00000000E5C88E1A",
  "manufacturer": "Sony UK",
  "memory": "1GB",
  "model": "3B",
  "processor": "BCM2837",
  "revision": "1.2",
  "gpioMask": "0x0FFFFFFC",
  "gpioMaskSerial": "0x0000C000",
  "date": "2021-06-06T13:10:32.000Z",
  "boot": "2021-06-06T09:03:31.860Z",
  "powerLed": 255,
  "load": 0.65,
  "temp": 52.6,
  "freq": 1200000000,
  "volt": 1.3062,
  "throttled": "0x00000000"
}
~ $
```

**Comment by ebaauw** [comment]
Damn.  I made an error while fixing the TypeError in v1.3.3.  Could you please try beta v1.3.4-0?

**Comment by fghekman** [comment]
The beta v1.3.4-0 is working for me.

**Comment by ebaauw** [comment]
Published v1.3.4 and removed 1.3.3.

**Comment by bgphoenix** [comment]
Confirmed to be working with 1.3.4. @ebaauw thank you good sir! 
PS: Your plugin is amazing to say the least
